openpaas-james/mailbox/encrypted/api/src/main/scala/com/linagora/openpaas/encrypted/EncryptedMailboxManager.scala

```diff
@@ -0,0 +1,485 @@
+package com.linagora.openpaas.encrypted
+
+import java.util.Optional
+import java.{lang, util}
+
+import javax.inject.Inject
+import org.apache.james.core.Username
+import org.apache.james.events.EventBus
+import org.apache.james.mailbox.MailboxManager.{MailboxCapabilities, MailboxRenamedResult, MessageCapabilities, SearchCapabilities}
+import org.apache.james.mailbox.exception.{MailboxException, MailboxExistsException, MailboxNotFoundException}
+import org.apache.james.mailbox.model.search.MailboxQuery
+import org.apache.james.mailbox.model.{Mailbox, MailboxACL, MailboxAnnotation, MailboxAnnotationKey, MailboxId, MailboxMetaData, MailboxPath, MessageId, MessageRange, MultimailboxesSearchQuery}
+import org.apache.james.mailbox.{MailboxManager, MailboxSession, MessageManager}
+import org.reactivestreams.Publisher
+import reactor.core.publisher.Flux
+
+class EncryptedMailboxManager @Inject()(mailboxManager: MailboxManager, eventBus: EventBus) extends MailboxManager {
+
+  def getEventBus(): EventBus = eventBus
```

Comment:
```suggestion
  def getEventBus(): EventBus = mailboxManager.getEventBus()
```

Reply:
only the StoreMailboxManager has the getEventBus() method.

Reply:
Then we do not need ` def getEventBus(): EventBus = eventBus` in the first place right?